test/test_encryption.py

```diff
@@ -212,11 +212,11 @@ def assertBinaryUUID(self, val):
 BASE = os.path.join(os.path.dirname(os.path.realpath(__file__)), "client-side-encryption")
 SPEC_PATH = os.path.join(BASE, "spec")
 
-OPTS = CodecOptions(uuid_representation=STANDARD)
+OPTS = CodecOptions()
```

Comment:
`uuid_representation=STANDARD` is no longer needed now that we decode uuid's to Binary by default.